SingularityBase/src/main/java/com/hubspot/singularity/SingularitySlaveUsage.java

```diff
@@ -78,6 +79,12 @@ public SingularitySlaveUsage(@JsonProperty("cpusUsed") double cpusUsed,
     this.slaveDiskTotal = slaveDiskTotal;
   }
 
+  @JsonIgnore
+  public boolean isOverloaded() {
+    // Any host where load5 is > 1 is overloaded. Also consider a higher threshold for load1 to take into account spikes large enough to be disruptive
+    return  systemLoad5Min > 1.0 || systemLoad1Min > 1.5;
```

Comment:
Do we want to make these values tunable knobs as well?

Reply:
yeah was going back and forth on that. Might as well for flexibility. I'll update 👍